packaging/fetch_requirements

```diff
@@ -45,6 +45,11 @@ MANAGER_INFO = {
         '/cloudify-management-worker',
     ]
 }
+STAGE_INFO = {'name': 'cloudify-stage', 'patterns': ['/cloudify-stage']}
+COMPOSER_INFO = {
+    'name': 'cloudify-blueprint-topology',
```

Comment:
Are you sure it's 'cloudify-blueprint-topology? It says above 'COMPOSER_INFO'

Reply:
Good catch its `cloudify-blueprint-composer` indeed. will update that